notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>Maybe add a bit more context/info about toggling between ES and self-vectorizing</p><p> </p><p>"...and Hugging Face models, letting you toggle between ES-vectorizing vs self-vectorizing."</p><p> </p><p>For notes, you can use callouts like this:</p>

<pre class="ql-syntax" spellcheck="false">&gt; [!TIP]
&gt; This notebook has been tested with ES 8.12.2.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks for the tip. </p>